### PR TITLE
Reviewing current promos

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/IY64Ykvn/3581-review-if-there-are-opportunities-for-ctas-on-high-traffic-website-pages

### Context

Now that there are no TTT events for a while, we should review where our promos are currently pushing people towards and consider adding more to pages currently without.

### CODE REVIEW

File: app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-itt.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Advice and support", classes: %w[ml-background]) do %>
+  <% end %>
+
+  <% promo.right_section(
+    heading: "Find out more about teacher training"
+  ) do %>
+      <p>Find out how you can start your teacher training, including top tips on writing a successful application.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-itt.html.erb:9]:
```suggestion
      <p>Learn how you can start your teacher training, including top tips on writing a successful application.</p>
```
less repetitive?

File: app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-no-degree.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Advice and support", classes: %w[ml-background]) do %>
+  <% end %>
+
+  <% promo.right_section(
+    heading: "Find out more about teaching"
+  ) do %>
+      <p>Find out what teaching is really like, including how you can get experience in the classroom to find out if it's the career for you.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-no-degree.html.erb:9]:
```suggestion
      <p>Learn what teaching is really like, including how you can get experience in the classroom to find out if it's the career for you.</p>
```
less repetitive? but you could probably word better here or tweak the heading!

File: app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-salaries.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Advice and support", classes: %w[ml-background]) do %>
+  <% end %>
+
+  <% promo.right_section(
+    heading: "Find out more about teaching"
+  ) do %>
+      <p>Find out more about the benefits of teaching, including what it's really like to be in the classroom.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-salaries.html.erb:9]:
```suggestion
      <p>Find out more about the benefits of teaching, including what it's really like to be in the classroom.</p>
```
can we make this less repetitive re Find out more?

File: app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-who-do-you-want-to-teach.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Advice and support", classes: %w[ml-background]) do %>
+  <% end %>
+
+  <% promo.right_section(
+    heading: "Find out more about teaching primary and secondary"
+  ) do %>
+      <p>Find out how you can get into teaching primary and secondary, including top tips on making a successful application.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-who-do-you-want-to-teach.html.erb:9]: Can we make this less repetitive re heading / body text?

**Comment by gemmadallmandfe** [comment]
Is it worth keeping the event promos (but not using them) @EllieNodder, given that we will want to bring them back for the comms burst?

**Comment by EllieNodder** [comment]
> Is it worth keeping the event promos (but not using them) @EllieNodder, given that we will want to bring them back for the comms burst?

Yeah - I think @ethax-ross got rid of them because something in them was stopping it from building? But think he said we can add them back in later if we want to use them later on

File: app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-itt.html.erb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Advice and support", classes: %w[ml-background]) do %>
+  <% end %>
+
+  <% promo.right_section(
+    heading: "Find out more about teacher training"
+  ) do %>
+      <p>Discover how you can start your teacher training, including top tips on writing a successful application.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-itt.html.erb:9]: Is 'start' a bit strong here?

File: app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-salaries.html.erb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Advice and support", classes: %w[ml-background]) do %>
+  <% end %>
+
+  <% promo.right_section(
+    heading: "Find out more about teaching"
+  ) do %>
+      <p>Explore more benefits of teaching, including what it's really like to be in the classroom.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-salaries.html.erb:9]: the first part and the second part of this sentence don't quite seem to go together - finding out what it's like to be in the classroom isn't a 'benefit of teaching'?

Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-salaries.html.erb:9]:
```suggestion
      <p>Explore more benefits of teaching, and learn what it's really like to be in the classroom.</p>
```

File: app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-video-interview-blog.html.erb [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-video-interview-blog.html.erb:1]: should this promo be deleted if we are not including blog promos in this PR?

File: app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-who-do-you-want-to-teach.html.erb [hunk 8]
```diff
@@ -0,0 +1,13 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Advice and support", classes: %w[ml-background]) do %>
+  <% end %>
+
+  <% promo.right_section(
+    heading: "Find out more about teaching primary and secondary"
+  ) do %>
+      <p>Explore how you can get into teaching primary and secondary, including top tips on making a successful application.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-who-do-you-want-to-teach.html.erb:9]:
```suggestion
      <p>Explore how you can get into teaching primary or secondary, including top tips on making a successful application.</p>
```

I wonder if we need to be clearer here that you don't do both?

File: app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-who-do-you-want-to-teach.html.erb [hunk 9]
```diff
@@ -0,0 +1,13 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Advice and support", classes: %w[ml-background]) do %>
+  <% end %>
+
+  <% promo.right_section(
+    heading: "Find out more about teaching primary and secondary"
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-who-do-you-want-to-teach.html.erb:7]:
```suggestion
    heading: "Find out more about teaching"
```
Do we need to say primary or secondary twice?

**Review by gemmadallmandfe — COMMENTED** [review]
Some very minor comments @EllieNodder - let's get this merged tomorrow (Thurs)!

File: app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-salaries.html.erb (hunk 6 above)
Comment (gemmadallmandfe) [reply app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-salaries.html.erb:9]: the first part and the second part of this sentence don't quite seem to go together - finding out what it's like to be in the classroom isn't a 'benefit of teaching'?

File: app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-video-interview-blog.html.erb (hunk 7 above)
Comment (EllieNodder) [reply app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-video-interview-blog.html.erb:1]: yep, on it

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**